src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/PartySvcRestClientImpl.java

```diff
@@ -73,4 +75,20 @@ public PartyDTO requestParty(SampleUnitDTO.SampleUnitType sampleUnitType, String
 
     return result;
   }
+
+  @Override
+  public SampleLinkDTO linkSampleSummaryId(String sampleSummaryId, String collectionExerciseId) throws RestClientException {
+    UriComponents uriComponents = restUtility.createUriComponents(appConfig.getPartySvc().getSampleLinkPath(), null, sampleSummaryId);
+    SampleLinkCreationRequestDTO sampleLinkCreationRequestDTO = new SampleLinkCreationRequestDTO();
+    sampleLinkCreationRequestDTO.setCollectionExerciseId(collectionExerciseId);
+    HttpEntity<SampleLinkCreationRequestDTO> httpEntity = restUtility.createHttpEntity(sampleLinkCreationRequestDTO);
+    ResponseEntity<SampleLinkDTO> responseEntity = restTemplate.exchange(uriComponents.toUri(), HttpMethod.PUT, httpEntity, SampleLinkDTO.class);
+
+    if (responseEntity != null && responseEntity.getStatusCode().is2xxSuccessful()) {
+      log.info("Created link Sample Summary Id: " + sampleSummaryId + "Collection exercise: " + collectionExerciseId);
+    } else {
+      log.error("Couldn't link Sample Summary Id: " + sampleSummaryId + "Collection exercise: " + collectionExerciseId);
```

Comment:
Might be worth logging the status code too and you'll want a space before "Collection exercise: ".